sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveSQLViewSuite.scala

```diff
@@ -72,9 +73,13 @@ class HiveSQLViewSuite extends SQLViewSuite with TestHiveSingleton {
           withTable("tab1") {
             (1 to 10).map(i => s"$i").toDF("id").write.saveAsTable("tab1")
 
-            // temporary view
-            sql(s"CREATE TEMPORARY VIEW tempView1 AS SELECT $tempFunctionName(id) from tab1")
-            checkAnswer(sql("select count(*) FROM tempView1"), Row(10))
+            // TODO: temporary function support for temporary view with sql text stored will
```

Comment:
nit: `TODO (SPARK-XXX): support temporary functions in temp views`

Reply:
yeah, but there are also test failure in udf-inner-join.sql related to temp function in temp views. I changed to another way to find these test cases. could you check again?
https://github.com/apache/spark/pull/30662/files#diff-ed19f376a63eba52eea59ca71f3355d4495fad4fad4db9a3324aade0d4986a47R118